src/test/java/edu/KevinChen/advancedjava/assignment1/SlingshotTest.java

```diff
@@ -0,0 +1,52 @@
+package edu.KevinChen.advancedjava.assignment1;
+
```

Comment:
make it so assignment one stuff (or code from other assignment) doesn't show up.